tests/slsa_analyzer/git_service/test_github.py

```diff
@@ -24,6 +24,6 @@ def test_is_detected(self) -> None:
         assert github.is_detected("git@github.com:7999/org/name")
         assert github.is_detected("ssh://git@github.com:7999/org/name")
         assert not github.is_detected("http://gitlab.com/org/name")
-        assert not github.is_detected("git@githubb.com:org/name")
-        assert not github.is_detected("git@not_supported_git_host.com:7999/org/name")
+        assert not github.is_detected("git@github0.com:org/name")
+        assert not github.is_detected("git@not-supported-git-host.com:7999/org/name")
```

Comment:
`_` isn't valid in .com dns names...

too many typos mask other problems....

fwiw, I have an additional commit not included in this PR that would add `github.example.com` to this block, as it's a more interesting check.